DependencyInjection/Configuration.php

```diff
@@ -84,19 +84,22 @@ public function getConfigTreeBuilder()
                                 ->arrayNode('publishable')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
+                                        ->scalarNode('form_tab')->defaultValue('form.tab_publish')->end()
+                                        ->scalarNode('form_group')->defaultValue('group_publish_workflow')->end()
                                     ->end()
                                 ->end()
                                 ->arrayNode('publish_time')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
+                                        ->scalarNode('form_tab')->defaultValue('form.tab_publish')->end()
+                                        ->scalarNode('form_group')->defaultValue('group_publish_workflow')->end()
```

Comment:
good thing. I removed it, because I first used `->with('TABNAME.GROUPNAME')` to override groups in tabs. In those cases, having 2 `form.` prefixes is ugly. But now I'm using `->tab('TABNAME')->with('GROUPNAME')`, it doesn't matter anymore. I'll revert this change